lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -415,6 +417,22 @@ def add_payment_method(xml, amount, payment_method, options)
         end
       end
 
+      def add_network_tokenization_card(xml, payment_method)
+        xml.paymentDetails do
+          xml.tag! 'EMVCO_TOKEN-SSL', 'type' => 'NETWORKTOKEN' do
+            xml.tokenNumber payment_method.number
+            xml.expiryDate do
+              xml.date(
+                'month' => format(payment_method.month, :two_digits),
+                'year' => format(payment_method.year, :four_digits)
+              )
+            end
+            xml.cryptogram payment_method.payment_cryptogram
+            xml.eciIndicator format(payment_method.eci, :two_digits)
+          end
+        end
+      end
```

Comment:
Does it make sense to rename this method to something like `add_payment_network_token` to align with the implementation of other gateways (https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/cyber_source.rb#L748)?

Reply:
Also, I see a number of gateways supporting the `supports_network_tokenization` method including [CyberSource](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/cyber_source.rb#L248) and others. The base [gateway.rb](https://github.com/activemerchant/active_merchant/blob/f54d1b6b32b9514d71b29bcc8fb0ad40aa7b9ff8/lib/active_merchant/billing/gateway.rb#L198) sets it to `false` by default. Should we add the same for worldpay? I'm not exactly sure how/where that method is used though.

cc: @shasum

Reply:
Thanks @jimilpatel24 The methods naming seems to be varied across the gateways eg [FirstData](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/firstdata_e4.rb) uses `add_network_tokenization_credit_card` while other gateways just include the token in  `add_payment_method` eg [NMI](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/nmi.rb). Wanted to name the method `add_network_tokenization_card` because when/if we enable apple_pay/google_pay for WorldPay, it will follow the same structure.
I am fine with changing it to `add_payment_network_token`, let me know your thoughts.

As for `supports_network_tokenization`, it doesn't seem to be used anywhere but it doesn't hurt to include it so I pushed in the latest commit.

Reply:
Sounds good. I will leave the method renaming decision upto you. I am fine either way.

Reply:
Standardisation may not offer a lot of benefit for private methods. A reasonably expressive name should be fine.

`supports_network_tokenization?` is a good callout 👍